css/animations.scss

```diff
@@ -227,12 +227,12 @@
         -khtml-opacity: 0;
         -webkit-opacity: 0;
         opacity: 0;
-        -o-transform: scale(0.96);
-        -ms-transform: scale(0.96);
-        -moz-transform: scale(0.96);
-        -khtml-transform: scale(0.96);
-        -webkit-transform: scale(0.96);
-        transform: scale(0.96);
+        -o-transform: scale(0.85);
```

Comment:
Re-define this animation in the component itself, so we don't change it for every component.